internal/config/config.go

```diff
@@ -130,10 +130,12 @@ type SQLKotlin struct {
 }
 
 type SQLPython struct {
-	EmitExactTableNames bool `json:"emit_exact_table_names" yaml:"emit_exact_table_names"`
-	Package   string     `json:"package" yaml:"package"`
-	Out       string     `json:"out" yaml:"out"`
-	Overrides []Override `json:"overrides,omitempty" yaml:"overrides"`
+	EmitExactTableNames bool       `json:"emit_exact_table_names" yaml:"emit_exact_table_names"`
+	EmitSyncQuerier     bool       `json:"emit_sync_querier" yaml:"emit_sync_querier"`
+	EmitAsyncQuerier    bool       `json:"emit_async_querier" yaml:"emit_async_querier"`
+	Package             string     `json:"package" yaml:"package"`
+	Out                 string     `json:"out" yaml:"out"`
+	Overrides           []Override `json:"overrides,omitempty" yaml:"overrides"`
```

Comment:
Do overrides work yet? If not, let's just remove them for now.

Reply:
They do work! I haven't extensively tested it, but it's working great to override a few jsonb fields with a custom type in a different project.